test_jsonify_stored/tests/common.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 ACSONE SA/NV.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests import SavepointCase
+
+
+class TestJsonifyMixin(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestJsonifyMixin, cls).setUpClass()
```

Comment:
do this always
```suggestion
        super(TestJsonifyMixin, cls).setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
and add `no_reset_password=True` if you create users.

Reply:
Yeah I still have this on my notes from your training session. Thanks for reminding me.